src/LiveComponent/assets/rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import { babel } from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
+export default [
+    {
+        input: 'src/live_controller.js',
+        output: {
+            name: 'LiveController',
+            file: 'dist/live-controller.esm.js',
+            format: 'esm',
+            sourcemap: false,
+        },
+        external: [
+            /@babel\/runtime/,
+            /core-js\//,
+            'stimulus',
+        ],
```

Comment:
In the _Rollup world_ we usually automate `external` by using the following code:
```js
import pkg from './package.json';

export default [
  {
    // ...
    external: Object.keys(pkg.dependencies)
  }
]
```

WDYT?

Reply:
I want to do whatever is standard :). But, the `@babel/runtime` needed to be a regex so that it matches all the individual, nested paths that are actually imported - like `@babel/runtime/...`. I got that from the note in this section - https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers

Reply:
Oh alright, I wasn't aware of that